pkg/kubelet/kubelet.go

```diff
@@ -317,6 +326,43 @@ func (kl *Kubelet) getPodContainerDir(podUID types.UID, ctrName string) string {
 	return newPath
 }
 
+// getTmpfsRootDir returns the full path to the kubelet's tmpfs root directory.
```

Comment:
I'm not sure we need all of these functions, nor do we really need the full matrix of storage media * reason.  If we agree where we want to go with it (some sort of characteristic matching), I can live with a purpose-built interface here.  Maybe just keep getTmpfsPodVolumeDir() and expose it as Host.GetTmpfsPodVolumeDir() with a TODO that it should become a more general interface.

That should buy time until we REALLY have to think about it, and still be pretty easy to convert.